plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -490,7 +504,7 @@ private KubernetesList convertToOpenShiftResources(KubernetesList resources) thr
             Controller controller = new Controller(client);
             Set<HasMetadata> oldEntities;
             try {
-                oldEntities = loadResources(client, controller, namespace, openshiftManifest, project, log);
+                oldEntities = loadResources(openshiftManifest);
```

Comment:
you can remove the local variables "namespace" and "controller" above as they were only used for this call.